src/OmniSharp.Roslyn.CSharp/Services/Signatures/SignatureHelpService.cs

```diff
@@ -133,7 +133,15 @@ private IEnumerable<IMethodSymbol> GetMethodOverloads(SemanticModel semanticMode
                 return new IMethodSymbol[] { };
             }
 
-            return symbol.ContainingType.GetMembers(symbol.Name).OfType<IMethodSymbol>();
+            var MethodOverloads = symbol.ContainingType.GetMembers(symbol.Name).OfType<IMethodSymbol>();
+            var BaseType = symbol.ContainingType.BaseType;
+            while(BaseType!=null && BaseType.ContainingNamespace.Name!= "System")
```

Comment:
spacing is off here. 
Also, perhaps it would be better to check for `baseTypeSymbol.SpecialType != SpecialType.System_Object` and terminate there instead rather than on "System" namespace

Reply:
The above mentioned change is failing for the case of attribute constructor as the System.Attribute class doesn't have the SpecialType as System_Object, hence for that case we will also get an Attribute.Attribute method which is not required.This is the reason the System Namespace has been used.